examples/convert_mri_and_trans.py

```diff
@@ -222,25 +240,22 @@
 # Writing defaced and anonymized FLASH MRI image
 # ----------------------------------------------
 #
-# Defacing the FLASH is a bit more complicated because it uses different
-# coordinates than the T1. Since, in the example dataset, we used the head
-# surface (which was reconstructed by freesurfer from the T1) to align the
-# digitization points, the points are relative to the T1-defined coordinate
-# system (called surface or TkReg RAS). Thus, you can you can provide the T1
-# or you can find the fiducials in FLASH voxel space or scanner RAS coordinates
-# using your favorite 3D image view (e.g. freeview). You can also read the
-# fiducial coordinates from the `raw` and apply the `trans` yourself.
-# Let's explore the different options to do this.
+# Defacing the FLASH works just like the T1 as long as they are aligned.
+
+# use ``trans`` to transform landmarks from the ``raw`` file to
+# the voxel space of the image
+landmarks = get_landmarks(
+    flash_mgh_fname,  # path to the FLASH scan
```

Comment:
```suggestion
    flash_fname,  # path to the FLASH scan
```
? for the same reason